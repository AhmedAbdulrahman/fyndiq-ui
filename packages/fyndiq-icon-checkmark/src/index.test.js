import React from 'react'
import { shallow } from 'enzyme'

import Checkmark from './'

describe('fyndiq-icon-brand', () => {
  it('should be rendered without props', () => {
    expect(shallow(<Checkmark />)).toMatchSnapshot()
  })
})
