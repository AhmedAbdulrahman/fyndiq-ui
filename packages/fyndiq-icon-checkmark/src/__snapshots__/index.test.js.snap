// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fyndiq-icon-brand should be rendered without props 1`] = `
<svg
  className=""
  fill="#999"
  height={15}
  viewBox="0 0 50 50"
  width={15}
>
  <path
    d="M16.8 42c-.4 0-1-.2-1.2-.5l-14-14.2c-.8-.7-.8-1.8 0-2.5.6-.7 1.7-.7 2.4 0l12.8 13 29-29c.7-.7 1.8-.7 2.5 0s.7 1.8 0 2.5L18 41.5c-.2.3-.7.5-1.2.5z"
  />
</svg>
`;
